src/ui/Controllers/OrganisationController.cs

```diff
@@ -82,19 +112,36 @@ public async Task<IActionResult> About(string ucasCode)
         [HttpPost]
         [Route("{ucasCode}/about")]
         public async Task<ActionResult> AboutPost(string ucasCode, OrganisationViewModel model)
+        {
+            var ucasInstitutionEnrichmentGetModel = await _manageApi.GetEnrichmentOrganisation(ucasCode);
+            var result = await SaveOrgansation(ucasInstitutionEnrichmentGetModel, model, ucasCode);
+            return result;
+        }
+
+        [HttpGet]
+        [Route("{ucasCode}/contact")]
+        public async Task<IActionResult> Contact(string ucasCode)
         {
             var ucasInstitutionEnrichmentGetModel = await _manageApi.GetEnrichmentOrganisation(ucasCode);
 
-            if (model.PublishOrganisation)
-            {
-                var result = await PublishOrgansation(ucasInstitutionEnrichmentGetModel, model, ucasCode);
-                return result;
-            }
-            else
-            {
-                var result = await SaveOrgansation(ucasInstitutionEnrichmentGetModel, model, ucasCode);
-                return result;
-            }
+            ucasInstitutionEnrichmentGetModel = ucasInstitutionEnrichmentGetModel ?? new UcasInstitutionEnrichmentGetModel { EnrichmentModel = new InstitutionEnrichmentModel() { AccreditingProviderEnrichments = new ObservableCollection<AccreditingProviderEnrichment>() } };
+
+            var enrichmentModel = ucasInstitutionEnrichmentGetModel.EnrichmentModel;
+            var ucasInsitution = await _manageApi.GetUcasInstitution(ucasCode);
+
+            var model = GetOrganisationViewModel(ucasCode, ucasInstitutionEnrichmentGetModel, ucasInsitution);
+            model.InstitutionName = (await _manageApi.GetOrganisations()).FirstOrDefault(x => x.UcasCode == ucasCode.ToUpperInvariant())?.OrganisationName;;
+
+            return View(model);
+        }
+
+        [HttpPost]
+        [Route("{ucasCode}/contact")]
+        public async Task<ActionResult> ContactPost(string ucasCode, OrganisationViewModel model)
+        {
+            var ucasInstitutionEnrichmentGetModel = await _manageApi.GetEnrichmentOrganisation(ucasCode);
+            var result = await SaveOrgansation(ucasInstitutionEnrichmentGetModel, model, ucasCode);
```

Comment:
nb. in a future version where we have a contactdetails editor, this would override the about your org fields with empties